src/components/Popover/PopoverArrow.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import cn from 'classnames';
+import * as RadixPopover from '@radix-ui/react-popover';
+
+import classes from './PopoverArrow.module.css';
+import { usePopoverContext } from './Context';
+export interface PopoverArrowProps {
+  children?: string;
+}
+export const PopoverArrow = ({ children }: PopoverArrowProps) => {
+  const { popoverVariant } = usePopoverContext();
+  return (
+    <RadixPopover.Arrow
+      className={cn(classes[`popover-arrow--${popoverVariant}`])}
```

Comment:
There seems to be some confusion about BEM naming convention. There should always be a "base" when using a "modifier" (modifier is any class with double dash in it).

This is a really great resource: https://9elements.com/bem-cheat-sheet/?ref=tiny-helpers to read up on the topic with some interactive functionality to help clear up what means what